ee/backend/app/src/main/kotlin/io/tolgee/ee/service/TranslationSuggestionServiceEeImpl.kt

```
@@ -0,0 +1,200 @@
+    keyService.checkInProject(key, project.id)
+    val language =
+      languageService.findEntity(languageId, project.id) ?: throw NotFoundException(Message.LANGUAGE_NOT_FOUND)
+
+    val normalizedTranslation =
+      try {
+        if (key.isPlural) {
+          normalizePlurals(mapOf("key" to dto.translation)).get("key") ?: ""
+        } else {
+          dto.translation
+        }
+      } catch (e: StringIsNotPluralException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)